migrations/portal/20220620152310-add_subscription.sql

```diff
@@ -0,0 +1,12 @@
+-- +migrate Up
+CREATE TABLE _portal_subscription (
+    id                          text PRIMARY KEY,
+    app_id                      text NOT NULL,
+    stripe_checkout_session_id  text NOT NULL,
```

Comment:
Make this nullable.

Reply:
I removed that column, see if it looks good to you?